.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Thanks for submitting a pull request! Before pressing that tempting green submit
```

Comment:
This is worded almost as a welcome message after submitting a pull request.  I think the template should follow more closely towards how we'd like to see an ideal pull request written.